geoviews/operation/projection.py

```diff
@@ -222,6 +220,55 @@ def _process_element(self, element):
         return element.clone(tuple(new_data[d.name] for d in element.dimensions()),
                              crs=self.p.projection)
 
+class project_vectorfield(_project_operation):
+
+    supported_types = [VectorField]
+
+    def _calc_angles(self, ut, vt):
+        # mathmetical convention
```

Comment:
```suggestion
        # mathematical convention
```
Maybe also worth noting that this is convention is chosen because of matplotlib if I recall correctly. 

Same with project_windbarbs.